src/main/java/frc/robot/RobotContainer.java

```diff
@@ -51,6 +62,17 @@ public RobotContainer() {
   private void configureBindings() {
     new Trigger(() -> triggerPressed())
         .whileTrue(new BasicDriveCommand(m_robotDrive, m_driverController));
+
+    // TODO: make actually intake under button press
+    new Trigger(this::buttonPressed)
+        .whileTrue(new RunIntake(m_intake));
+  }
+
+  public boolean buttonPressed() {
+    if (m_operatorController.getRawButton(Bindings.intakeNote)) {
```

Comment:
STUDENTS! Stop using if statements if you're returning a boolean...

Cease this:
```java
if (condition) {
	return true;
}
else {
	return false;
}
```

Be a good programmer. Be responsible. Do this instead:
```java
return condition;
```

Reply:
Responded to